packages/Combobox/src/combobox.scss

```diff
@@ -131,6 +131,11 @@
     &--active,
     &.active {
```

Comment:
I think we need to specify a white background color for active. If not, we end up with a grey background when the menu is open:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/1ec0d6bb-0abc-40f9-ba11-f5c1b69a6a2e)
This is taken for the select, but it's the same for the combobox.